ci/script.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+source ./tempEnv.sh
+
+set -eEuo pipefail
+
+echo "=== Local variables ==="
+ENABLE_LOGS="${ENABLE_LOGS:-true}"
+ALLOW_AUDIT_FAILURES="${ALLOW_AUDIT_FAILURES:-false}"
+ALLOW_TEST_FAILURES="${ALLOW_TEST_FAILURES:-false}"
```

Comment:
Where do we use these vars :thinking:

Reply:
Those are usually used in the test step. As there are no tests run in this pipeline those env vars can be removed